configs/development.js

```diff
@@ -10,6 +10,15 @@ const gitInfo = GitUrlParse(process.env.GATSBY_SOURCE)
 
 module.exports = {
   plugins: [
+    {
```

Comment:
Is the plan to run markdown cleaner on production builds as well?

Reply:
Oops, knew something was missing. I'll add it to the production side as well.

Thinking out loud here about framing this two-part effort for md-to-mdx conversion: 

1. This plugin is part one. Its purpose is to prevent parliament builds (develop and production) from breaking. It does it by changing the local/copied markdown used for the build.
2. The markdown-cleaner CLI addition is part two. Its purpose should be aimed at the source docs to either warn users of the problems or fix them as desired. See @icaraps suggestion above: https://github.com/adobe/parliament-client-template/pull/36#issuecomment-712709132

Got it. Now I think I'm pretty clear now about the purpose of these two different solutions along with where and how they should be applied.